frontend/src/components/employees/employees.module.scss

```diff
@@ -0,0 +1,50 @@
+.employees {
+  grid-area: main;
+  display: grid;
+  margin: 0 1em;
+  grid-template:
+    'header addbutton' 5em
+    'table table' auto
+    / auto 10em;
+}
+
+h1 {
+  grid-area: header;
+  font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
+}
+
+.addButton {
+  grid-area: addbutton;
+  margin: 2em;
+}
+
+Table {
+  grid-area: table;
+}
+
+.editbtn,
+.removebtn {
+  text-align: right;
+  width: 1em;
+}
+
+.editButton,
+.removeButton {
+  border: none;
+  outline: none;
+  background-color: transparent;
+  font-weight: 100;
```

Comment:
font-weight seems like something we might reuse in other parts of the app for consistency? If so maybe we should move this to constants too?

Reply:
This one is there to make the pencil icon look right. If we choose to use material UI icons it will no longer be needed.